### PR TITLE
Docs for NetSuite bulk actions

### PR DESCRIPTION
**Pull request by zweedao**

### CODE REVIEW

**Comment by zweedao** [comment]
@graceliching can you help review

**Comment by zweedao** [comment]
ok separated @graceliching

File: connectors/netsuite/action-batch-upsert.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+They accept a list of a maximum 200 records. Actions will fail if more than 200 records are provided.
+
+## Input fields
+
+Select the `Standard record` or `Custom record` you want to upsert. In this example, we select `Customer`:
+
+![Select Netsuite record](/assets/images/connectors/netsuite/standard-record.png)
+
+After that configure these sections:
```
Comment (graceliching) [inline connectors/netsuite/action-batch-upsert.md]: I think we can remove these "configure below:" statements. They're not complete sentences and are kind of strange to end the section.

File: connectors/netsuite/action-batch-update.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+They accept a list of a maximum 200 records. Actions will fail if more than 200 records are provided.
+
+## Input fields
+
+Select the `Standard record` or `Custom record` you want to update. In this example, we select `Customer`:
+
+![Select Netsuite record](/assets/images/connectors/netsuite/standard-record.png)
+
+After that configure these sections:
```
Comment (graceliching) [inline connectors/netsuite/action-batch-update.md]: I think we can remove these "configure below:" statements. They're not complete sentences and are kind of strange to end the section.

File: connectors/netsuite/action-batch-create.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+The actions `Create standard records in bulk` and `Create custom records in bulk` allow you to create in bulk standard records or custom records.
+
+They accept a list of a maximum 400 records. Actions will fail if more than 400 records are provided.
+
+## Input fields
+Select the `Standard record` or `Custom record` you want to create. In this example, we select `Customer`:
+
+![Select Netsuite record](/assets/images/connectors/netsuite/standard-record.png)
+
+After that configure these sections:
```
Comment (graceliching) [inline connectors/netsuite/action-batch-create.md]: I think we can remove these "configure below:" statements. They're not complete sentences and are kind of strange to end the section.

File: connectors/netsuite/action-batch-create.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+In this example, we map the data from the Box trigger `New CSV file in directory` into NetSuite record `Customer`. Notice that only the shortlisted fields `First name`, `Last name`, `Email`, `phone` are displayed here.
+
+![Field mappings](/assets/images/connectors/netsuite/batch-mapping.png)
+
+The followings are all input fields in this section:
+
+| Field name | Description |
+|---|---|
+| Record source list | You need to provide a [list data pill](https://docs.workato.com/features/list-management.html) here. For example, list of rows in a CSV file. The action will automatically iterate through all items in the list and map them to NetSuite records. |
```
Comment (graceliching) [inline connectors/netsuite/action-batch-create.md]: The link can refer to the docs within docs /features/list-management

File: connectors/netsuite/action-batch-create.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+| Record fields | Here you need to map the data pills under the list data pill provided above. For example, CSV column names under the `CSV Rows` list data pill. The action will automatically iterate through all items in the list and map them to NetSuite records. |
+
+## Output fields
+
+| Field name | Description |
+|---|---|
+| Number of processed records | Total number of records processed. |
+| Number of successful records | Number of records successfully created. |
+| Number of failed records | Number of records failed to be created. |
+| Successful records | A list data pill containing all successful records. You can use this to [iterate through all records in the list](https://docs.workato.com/features/list-management.html). The data fields of each record is shown below: |
```
Comment (graceliching) [inline connectors/netsuite/action-batch-create.md]: same here

File: connectors/netsuite/action-batch-create.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+| - Internal ID | Internal ID of this record. |
+| - External ID | External ID of this record. |
+| - Record | Contains all data fields of this record. |
+| - List size | Number of successful records in this list. |
+| Failed records | A list data pill containing all failed records. You can use this to [iterate through all records in the list](https://docs.workato.com/features/list-management.html). The data fields of each record is shown below: |
```
Comment (graceliching) [inline connectors/netsuite/action-batch-create.md]: same for all other links

File: connectors/netsuite/action-batch-create.md (hunk 3 above)
Comment (zweedao) [reply connectors/netsuite/action-batch-create.md]: done, edited a bit

File: connectors/netsuite/action-batch-create.md (hunk 4 above)
Comment (zweedao) [reply connectors/netsuite/action-batch-create.md]: done

File: connectors/netsuite/action-batch-create.md (hunk 5 above)
Comment (zweedao) [reply connectors/netsuite/action-batch-create.md]: done

File: connectors/netsuite/action-batch-create.md (hunk 6 above)
Comment (zweedao) [reply connectors/netsuite/action-batch-create.md]: done

File: connectors/netsuite/action-batch-update.md (hunk 2 above)
Comment (zweedao) [reply connectors/netsuite/action-batch-update.md]: done, edited

File: connectors/netsuite/action-batch-upsert.md (hunk 1 above)
Comment (zweedao) [reply connectors/netsuite/action-batch-upsert.md]: done, edited

**Comment by zweedao** [comment]
done @graceliching

**Pull request merged**